redi-connect-front/src/pages/app/me/Me.tsx

```diff
@@ -29,187 +41,163 @@ export type SignUpFormType =
+
+      {saveResult === 'error' && <><br/><br/><br/>An error occurred, please try again.</>}
+      {saveResult === 'success' && <>Your profile was saved.</>}
       {!loading &&
         <>
-          <FullScreenCircle loading={saveResult === 'submitting'} />
-          {/* <Button
-            onClick={() => {
-              formik.handleSubmit()
-            }}
-            disabled={saveResult === 'submitting'}
-          >
-                  Save
-          </Button> */}
-          {/*
-          <form onSubmit={e => e.preventDefault()}> */}
+          {saveResult === 'submitting' && 'part of the page loading...'}
+
+          <Columns>
+            <Columns.Column
+              size={3}
+            >
+              <div className="file is-boxed">
+                {/* SHOWING THE AVATAR
+                 <Avatar s3Key={profileAvatarImageS3Key} /> */}
+                <label className="file-label" htmlFor="avatar-upload">
+                  <span className="file-cta">
+                    <span className="file-label">Upload file</span>
+                  </span>
+                  <ReactS3Uploader
+                    name="avatar-upload"
+                    id="avatar-upload"
+                    className="file-input"
+                    signingUrl={S3_UPLOAD_SIGN_URL}
+                    accept="image/*"
+                    uploadRequestHeaders={{ 'x-amz-acl': 'public-read' }}
+                    // preprocess={onUploadStart}
+                    onSignedUrl={(c: any) => console.log(c)}
+                    onError={(c: any) => console.log(c)}
+                    onFinish={onUploadSuccess}
+                    contentDisposition="auto"
+                    // inputRef={(cmp: any) => setUploadInput(cmp)}
+                  />
+                </label>
+              </div>
+
+            </Columns.Column>
+
+            <Columns.Column
+              size={9}
+            >
+              <Heading
+                size={1}
+                responsive={{ mobile: { textSize: { value: 2 } } }}
+                weight="normal"
+                renderAs="h1"
+                spaced={true}
+              >
+                Hi, {profile.firstName}
+              </Heading>
+
+            </Columns.Column>
+          </Columns>
+
           <About />
           <Mentoring />
-          {/* <Step2Background type={profile.userType} {...formik} />
-            <Step3Profile type={profile.userType} {...formik} />
-            <Step4ContactData type={profile.userType} {...formik} />
-            <Step5Categories type={profile.userType} {...formik} /> */}
-          {/* </form> */}
-
-          {saveResult === 'error' && <><br/><br/><br/>An error occurred, please try again.</>}
-          {saveResult === 'success' && <>Your profile was saved.</>}
+          <Columns>
+            <Columns.Column
+              size={6}
+            >
+              <Contacts/>
+            </Columns.Column>
+
+            <Columns.Column
+              size={6}
```

Comment:
Ugh, is this a result of our lint configuration? Can we get the entire tag on one line?

Reply:
Perhaps let's override our eslint config to make this fit onto one line.. much easier to read

Reply:
This is the endre linter most possibly :D

Reply:
lol